src/frontend-scripts/components/section-main/DisplayLobbies.jsx

```diff
@@ -329,13 +330,41 @@ const DisplayLobbies = props => {
 
 			if (player.customCardback && (!userInfo.userName || !(userInfo.userName && userInfo.gameSettings && userInfo.gameSettings.disablePlayerCardbacks))) {
 				total.push(
-					<div key={total.length} className={classes} data-tooltip={player.userName} data-inverted="">
+					<div
+						key={total.length}
+						className={classes}
+						data-tooltip={player.userName}
+						data-inverted=""
+						style={(() => {
+							const style = {};
+
+							if (player && player.staffRole === 'editor') {
+								style[`box-shadow`] = `0 0 5px 2px ${player.staffEditorCustomColour || '#05bba0'}`;
+							}
+
+							return style;
+						})()}
+					>
 						<img src={`../images/custom-cardbacks/${player.userName}.${player.customCardback}?${player.customCardbackUid}`} />
 					</div>
 				);
 			} else {
 				total.push(
-					<div key={total.length} className={classes} data-tooltip={player.userName} data-inverted="">
+					<div
+						key={total.length}
+						className={classes}
+						data-tooltip={player.userName}
+						data-inverted=""
+						style={(() => {
+							const style = {};
+
+							if (player && player.staffRole === 'editor') {
+								style[`box-shadow`] = `0 0 5px 2px ${player.staffEditorCustomColour || '#05bba0'}`;
+							}
+
+							return style;
+						})()}
+					>

```

Comment:
this is basically a duplicate of the `if` part of the statement... why not use the same div for both of them, and use a ternary where they differ?